book/types_of_data.md

```diff
@@ -42,7 +42,7 @@ A string of characters that represents text. There are a few ways these can be c
   - `$"6 x 7 = (6 * 7)"`
   - `ls | each { |it| $"($it.name) is ($it.size)" }`
 - Bare strings
-  - `echo hello`
+  - `hello`
```

Comment:
I think this would be evaluated as a command first. Bare strings are relevant in the arguments or as part of a data structure.

Reply:
Thanks!